### ISSUE TITLE
 Results on adpatation from PASCAL VOC to Clipart Dataset does not match 

### ISSUE THREAD

**Issue by isalirezag** [naoto0804/cross-domain-detection#15]
I am trying to get the results in table 3 for Baseline  but it does not match with the paper, can you provide more details on the specifics please.

1) For training on the baseline you mentioned to use VOC2007 and 2012, should i convert them via cyclegan first or baseline means just training on original VOC2007 + 2012  images?

2) also you mentioned in the paper that:
`For the target-domain images, the ones with
instance-level annotations were used when discussing the
performance gap between our methods and the ideal case
quantitatively. The target-domain images were split into
training set and test set by a ratio of 1:1. For the training set, the bounding box information was ignored to meet
the proposed situation.`
can you please clarify what do you mean by: ` the ones with
instance-level annotations were used when discussing the
performance gap between our methods and the ideal case
quantitatively. `

3) when I download the clipart data set it includes test and train, are the results reported on the paper for baseline computed by training on  VOC2007 + 2012 and test on test images in clipart (500 images) or test on the total images in the clipart (1000 images)

Thank you!

**Comment by naoto0804** [comment]
Hi, thank you for your interest in our paper.

1. the latter, I just used the pre-trained SSD300 model provided by ChainerCV.

2. Hmm, do you know *target only* or *train on target* methods used in domain adaptation for classification (e.g., https://arxiv.org/abs/1702.05464)? All the description is analogous to that.

3. The former, for fair comparison among all the methods discussed.
